src/xz/file_io.c

```diff
@@ -933,7 +957,11 @@ io_open_dest_real(file_pair *pair)
 #ifndef TUKLIB_DOSLIKE
 		flags |= O_NONBLOCK;
 #endif
+#ifdef _MSC_VER
+		const mode_t mode = _S_IREAD | _S_IWRITE;
```

Comment:
Perhaps it's fine to use plain `int` here as that is what `_open` needs. Then there is no need to `typedef int mode_t` at all. The only other `mode_t` occurrence isn't used on Windows.

Reply:
Fixed in a new commit. Removed typedef `mode_t` and simply use `int` instead.